docs/protocol.md

```diff
@@ -326,142 +326,107 @@ a subsequent visit) some $passphrase$:
+| -------------------------------------------------------------------------------------------------------------------- | ------------------------------- | ----------------------------------------------------------------------------- |
+|                                                                                                                      | $\longrightarrow$ `RequestKeys` |                                                                               |
+|                                                                                                                      |                                 | $`pks = \{(pk_{R,i}^{APKE}, pk_{R,i}^{PKE}, pk_{R,i}^{fetch})\}`$ for all $i$ |
+|                                                                                                                      | $`pks \longleftarrow`$          |                                                                               |
+| **TODO:** verification per $NR$ and $J_i$                                                                            |                                 |                                                                               |
+|                                                                                                                      |                                 |                                                                               |
+| **Reply case:** The journalist replaces their own keys with those of the source to whom they are replying:           |                                 |                                                                               |
+| $`pks \gets pks \setminus \{pk_S^{APKE}, pk_S^{PKE}, pk_S^{fetch}\} \cup \{pk_R^{APKE}, pk_R^{PKE}, pk_R^{fetch}\}`$ |                                 |                                                                               |
+
+### 6. Sender submits a message <!-- Figure 1 as of 7703a58 -->
+
+Then, for some message $m$, for all keys $(pk_{R,i}^{APKE}, pk_{R,i}^{PKE},
+pk_{R,i}^{fetch}) \in pks$:
+
+| Source                                                                              |                                 | Server                                         |
+| ----------------------------------------------------------------------------------- | ------------------------------- | ---------------------------------------------- |
+| $`pt \gets m \Vert pk_S^{fetch} \Vert pk_S^{PKE} `$                                 |                                 |                                                |
+| $`ct^{APKE} \gets \text{SD-APKE.AuthEnc}(sk_S^{APKE}, pk_{R,i}^{APKE}, pt, NR, -)`$ |                                 |                                                |
+| $`ct^{PKE} \gets \text{SD-PKE.Enc}(pk_{R,i}^{PKE}, pk_S^{APKE}, -, -)`$             |                                 |                                                |
```

Comment:
I agree that this is what it says in the manuscript - just adding a reference comment for our clarity that the thing that is encrypted here is just a sender dh-akem pubkey, which is part of $pk_S^{APKE}$. The reason it's represented this way is that in Tamarin that is modeled as one single key. In practice it is (dh-akem pubkey, pq kem psk pubkey).   The dh-akem pubkey is used by the recipient to auth-decrypt the ciphertext.

When journalists are replying to sources, the $pk_S^{APKE}$ referred to here is their long-term dhakem reply key. It doesn't have a long term pq psk kem counterpart.